### DISCUSSION TITLE
Any Open source Git hub repo to run my web app in electron desktop app?

### DISCUSSION THREAD

**Discussion by wshahzad777** [community/community#134484] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I need to run my web app in electron app, is there any open source basic setup to run web app, create package, and build of app , so that i can use it on my desktop

**Comment by QaisarMuhammad** [comment, marked as answer]
Yes you can use this repo, just clone it, follow the instructions to setup, change your project URL as suggested to run your web application
https://github.com/QaisarMuhammad/Electron-app

**Reply by wshahzad777** [reply]
ok thank you for your help!!!

**Comment by LiteBrite82** [comment]
Hi @wshahzad777 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
